### DISCUSSION TITLE
v2fly.org官网上说，VLESS 已被弃用并且可能被移除，是时候准备后路了么？

### DISCUSSION THREAD

**Discussion by cumcw97u4nba3ykv** [v2fly/v2ray-core#2110] [General]
[https://www.v2fly.org/v5/config/proxy/vless.html](url)

> VLESS
> VLESS 是一个无状态的轻量传输协议，它分为入站和出站两部分，可以作为 V2Ray 客户端和服务器之间的桥梁。

> TIP
> VLESS 没有自带加密，请用于可靠信道，如 TLS。

> WARNING
> VLESS 已被弃用并且可能被移除。
> 请考虑使用 Trojan 作为替代品。

**Comment by Septrum101** [comment]
用trojan平替。现在支持v2ray已经可以用wss等所有transport来驱动trojan

**Comment by hjm1a** [comment]
_澄清一下，VLESS不是v2ray首创的，是v2ray的一个社区分支Xray先推出的，而且仍然受Xray官方支持，sing-box也没有弃用VLESS。_
所以，简而言之：**使用VLESS的，更换core为xray或者sing-box，这两个core都没有对VLESS停止支持。**
